pennylane/measurements.py

```diff
@@ -270,6 +270,11 @@ def shape(self, device=None):
             # qml.sample() case
             return (1, device.shots, len_wires)
 
+        if self.return_type == Shadow:
+            # the first entry of tensor represents the measured bits,
+            # and the second indicate the indices of the unitaries used
+            return (2, device.shots, len(device.wires))
```

Comment:
Fine as is, but it makes me wonder if we should just define `ShadowMeasurementProcess.shape`.

Reply:
I agree entirely.